patchwork-extensions/src/main/java/com/patchworkmc/mixin/extension/MixinItem.java

```diff
@@ -60,6 +67,24 @@ private void onConstruct(Item.Settings settings, CallbackInfo info) {
 		toolClasses.putAll(extension.getToolClasses());
 	}
 
+	@Redirect(method = "isEnchantable(Lnet/minecraft/item/ItemStack;)Z", at = @At(value = "INVOKE", target = "Lnet/minecraft/item/Item;getMaxCount()I"))
+	private int getMaxCountForStack(Item item, ItemStack stack) {
+		return getItemStackLimit(stack);
+	}
+
+	@ModifyConstant(method = "rayTrace(Lnet/minecraft/world/World;Lnet/minecraft/entity/player/PlayerEntity;Lnet/minecraft/world/RayTraceContext$FluidHandling;)Lnet/minecraft/util/hit/HitResult;", constant = @Constant(doubleValue = 5.0D))
+	private static double modifyReachDistance(double originalDist, World world, PlayerEntity player, RayTraceContext.FluidHandling fluidHandling) {
+		// TODO: return player.getAttributeInstance(REACH_DISTANCE).getValue();
```

Comment:
Is this part of IForgeItem? It looks like something separate. This is probably more appropriate for a different module.